evalml/pipelines/utils.py

```diff
@@ -166,6 +166,7 @@ def _get_preprocessing_components(
 
 def _get_pipeline_base_class(problem_type):
     """Returns pipeline base class for problem_type."""
+    problem_type = handle_problem_types(problem_type)
```

Comment:
Nope--but only because it's a private method that we use in tests 😂 

Just happened to be that everywhere where we used it, we used the ProblemTypes enum so this didn't cause problems before but I noticed it since I tried using strings!